ofmeet/src/web/ofmeet-uisettings.jsp

```diff
@@ -148,6 +158,7 @@
             JiveGlobals.setProperty( "org.jitsi.videobridge.ofmeet.default.remote.displayname", defRemoteDisplName );
             JiveGlobals.setProperty( "org.jitsi.videobridge.ofmeet.default.speaker.displayname", defDomSpkrDisplName );
             JiveGlobals.setProperty( "org.jitsi.videobridge.ofmeet.default.local.displayname", defLocalDisplName );
+            JiveGlobals.setProperty( "org.jitsi.videobridge.ofmeet.film.strip.max.height", filmstripMaxHeight );            
```

Comment:
lets move this to ofmeetconfig too. Lazy as I am, I didn't do everything yet, but I at least try to create all new properties, and everything that I modify for whatever reason, there.